spec/requests/v6/assessments_controller/simple_spec.rb

```diff
@@ -1142,9 +1228,9 @@ def cash_transactions(amount)
           it "logs redacted dependants dob" do
             expect(partner_log.fetch(:dependants))
               .to eq(
-                [{ date_of_birth: "2014-06-06", in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 },
-                 { date_of_birth: "2013-06-06", in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 },
-                 { date_of_birth: "2004-06-06", in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 }],
+                [{ date_of_birth: date_of_birth_1, in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 },
+                 { date_of_birth: date_of_birth_2, in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 },
+                 { date_of_birth: date_of_birth_3, in_full_time_education: true, relationship: "child_relative", monthly_income: 0, assets_value: 0.0 }],
```

Comment:
This looks wrong - it now obfuscates the test intention (that the DOBs are clearly visible as redacted)